sdk/src/Core/AWSXRayRecorder.Core.csproj

```diff
@@ -38,8 +38,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="AWSSDK.Core" Version="3.3.25.1" />
-    <PackageReference Include="AWSSDK.XRay" Version="3.3.3" />
+    <PackageReference Include="AWSSDK.Core" Version="3.5.1.23" />
```

Comment:
As discussed, can we provide a range of version here as `Version="[3.3.25.1,4.0)`

Reply:
Updated